src/pizza.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/local/bin/python -i
+#!/usr/bin/env python -i
```

Comment:
@sjplimp No, there is no portable way to do it that I'm aware of. IMHO this is going the wrong way since I would see pizza as an add-on package that is loaded after launching the interpreter.